open-apparel-registry/open-apparel-registry
# Fix VectorGrid bug

## Overview

We use two layers to display facilities in the map - the
VectorTileFacilitiesLayer and the VectorTileFacilityGridLayer.
The VectorTileFacilityGridLayer is the 'dots' on the map, and
is shown when the map is zoomed out to a certain level.

When the map was zoomed in far enough that the grid is hidden,
and the user navigated away from the map, an error was being thrown.
By only rendering the grid layer when it's visible, that error is
prevented.

Connects #1563 

## Testing Instructions

On `develop`:
* Run `./scripts/server` and navigate to a facility details page.
* Navigate to the login page or dashboard (or anywhere in the site away from the map).
* You will see an error message. 

On this branch:
* Repeat the steps above, and you should not see an error message.
* Zoom in and out on the map and confirm that the dots appear and disappear at the appropriate level, and in general works the same as before. 

## Checklist
